containers/ddev-php-base/Dockerfile

```diff
@@ -84,7 +84,7 @@ http://nginx.org/packages/debian `lsb_release -cs` nginx" > /etc/apt/sources.lis
 # Install mariadb_repo_setup to get mariadb-client from them directly
 RUN curl -LsSf https://r.mariadb.com/downloads/mariadb_repo_setup -o /usr/local/bin/mariadb_repo_setup && chmod +x /usr/local/bin/mariadb_repo_setup
 # Search for CHANGE_MARIADB_CLIENT to update related code.
-RUN /usr/local/bin/mariadb_repo_setup --mariadb-server-version="mariadb-10.11"
+RUN /usr/local/bin/mariadb_repo_setup --mariadb-server-version="mariadb-10.11" --skip-maxscale --skip-tools
```

Comment:
This is fine, but why would we install it in ddev-php-base anyway, I wonder.  Seems like it would make more sense just in ddev-webserver.

Reply:
I think it was only added to `ddev-php-base` because `apt-get install mariadb-client` was there in the first place.

I agree that it's better to have it in `ddev-webserver` because it's so tedious to wait for the Docker image to be ready and who knows how many times we'll need to change it in the future.

I will move it completely to `ddev-webserver`.